### PR TITLE
Use node 14 and npm7

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       matrix:
         php-versions: ['7.3', '7.4']
```
Comment (skjnldsv) [inline .github/workflows/lint.yml:16]: 8?

**Review by skjnldsv — APPROVED** [review]

File: .github/workflows/lint.yml (hunk 1 above)
Comment (nickvergessen) [reply .github/workflows/lint.yml:16]: For another time

**Comment by nickvergessen** [comment]
Ah crap, dependabot merged tonight....

**Comment by skjnldsv** [comment]
Lots of missing protection checks :)

**Comment by nickvergessen** [comment]
> Lots of missing protection checks :)

Added one of all

**Pull request merged**